app/notifiers/notifier/base.rb

```diff
@@ -0,0 +1,58 @@
+# The base class of notifiers. This is meant to be called by the Notifications Framework
+#
+# @api notifications
+class Notifier::Base
+  include ApplicationNotificationsConcern
+
+  class << self
+    # This is to allow client code to call create notifications without explicitly creating
+    # notifiers.
+    #
+    # @param [Symbol] symbol The symbol which is the name of a method
```

Comment:
I think you should remove this line(the @param documentation), this method is not called by user, document params won't help.